File: civgenesis-core/src/main/java/io/github/cuihairu/civgenesis/core/protocol/ProtocolFlags.java
package io.github.cuihairu.civgenesis.core.protocol;

public final class ProtocolFlags {
    private ProtocolFlags() {}

    public static final long ERROR = 0x01;
    public static final long COMPRESS = 0x02;
    public static final long ENCRYPT = 0x04;
    public static final long ACK_REQUIRED = 0x08;
}

File: clients/unity/CivGenesis.Client/Protocol/ProtocolFlags.cs
namespace CivGenesis.Client.Protocol
{
    public static class ProtocolFlags
    {
        public const ulong Error = 0x01;
        public const ulong Compress = 0x02;
        public const ulong Encrypt = 0x04;
        public const ulong AckRequired = 0x08;
    }
}

